python/bencode.py

```diff
@@ -92,16 +88,16 @@ def iced_bencode_decode(s):
     '__FAILED__'
     """
     result = []
-    rest = s
+    s = s.encode('utf-8')
```

Comment:
NIT: Is `b` better?
e.g. `b = s.encode('utf-8')`

Reply:
Oh yes, I forgot it here.